core/src/main/scala/chisel3/Data.scala

```diff
@@ -157,6 +157,26 @@ package experimental {
       target.direction
     }
 
+    private def isBinding[B <: ConstrainedBinding : Manifest](target: Data) = {
+      target.topBindingOpt match {
+        case Some(b: B) => true
+        case _ => false
+      }
+    }
+
+    /** Check if a Chisel type is an IO port
+      * @param x Chisel type.
+      */
```

Comment:
```suggestion
    /** Check if a given `Data` is an IO port
      * @param the `Data` to check
      * @return `true` if x is an IO port, `false` otherwise
      */
```
Some minor tweaks, feels a bit obvious but don't want to say "Chisel type" because that's ambiguous, just stick with `Data`.

Reply:
desn't `@param` still need the name of the parameter?

`@param x the `Data` to check`